go/ct/spc/specification.go

```diff
@@ -1347,6 +1347,34 @@ func getAllRules() []Rule {
 
 	rules = append(rules, callOp()...)
 
+	// --- SELFDESTRUCT ---
+
+	//                                                 revision   warm  destColdCost accCreatFee  hasSelfDestructed
+	rules = append(rules, nonStaticSelfDestructRules(R07_Istanbul, false, vm.Gas(0), vm.Gas(25000), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R07_Istanbul, false, vm.Gas(0), vm.Gas(25000), false)...)
+	rules = append(rules, nonStaticSelfDestructRules(R07_Istanbul, true, vm.Gas(0), vm.Gas(0), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R07_Istanbul, true, vm.Gas(0), vm.Gas(0), false)...)
+	rules = append(rules, nonStaticSelfDestructRules(R09_Berlin, false, vm.Gas(2600), vm.Gas(25000), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R09_Berlin, false, vm.Gas(2600), vm.Gas(25000), false)...)
+	rules = append(rules, nonStaticSelfDestructRules(R09_Berlin, true, vm.Gas(0), vm.Gas(0), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R09_Berlin, true, vm.Gas(0), vm.Gas(0), false)...)
+	rules = append(rules, nonStaticSelfDestructRules(R10_London, false, vm.Gas(2600), vm.Gas(25000), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R10_London, false, vm.Gas(2600), vm.Gas(25000), false)...)
+	rules = append(rules, nonStaticSelfDestructRules(R10_London, true, vm.Gas(0), vm.Gas(0), true)...)
+	rules = append(rules, nonStaticSelfDestructRules(R10_London, true, vm.Gas(0), vm.Gas(0), false)...)
```

Comment:
With additional revisions this will become longer and longer ... at some point we should produce those cases using nested loops. But I guess we can leave this until we add the Paris revision.